opsmngr/backup_configs_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestAutomation_GetBackupConfig(t *testing.T) {
+	client, mux, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc(fmt.Sprintf("/groups/%s/backupConfigs/%s", projectID, clusterID), func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+		_, _ = fmt.Fprint(w, `{
+			  "clusterId" : "1",
+			  "encryptionEnabled" : false,
+			  "groupId" : "1",
+			  "SSLEnabled" : false,
+			  "statusName" : "STARTED",
+			  "storageEngineName" : "WIRED_TIGER"
+}`)
+	})
+
+	config, _, err := client.BackupConfigs.Get(ctx, projectID, clusterID)
+	if err != nil {
+		t.Fatalf("BackupConfigs.Get returned error: %v", err)
+	}
+
+	expected := &BackupConfig{
+		GroupID:           "1",
+		ClusterID:         "1",
+		StatusName:        "STARTED",
+		StorageEngineName: "WIRED_TIGER",
+		EncryptionEnabled: false,
+		SSLEnabled:        false,
+	}
+	if diff := deep.Equal(config, expected); diff != nil {
+		t.Error(diff)
+	}
+}
+
+func TestAutomation_ListBackupConfig(t *testing.T) {
```

Comment:
same as above, the test method name should match the Struct, underscore, function pattern